### PR TITLE
MediaPipe HandPose Detection using Python

### LINKED ISSUE #4794: [Machine Learning] Handpose Detection using MediaPipe and React.js
#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Handpose Detection using MediaPipe and React.js

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

Handpose recognition is a deep learning technique that allows you to detect different points on your hand. These points on your hand are commonly referred to as landmarks. These landmarks consist of joints, tips, and bases of your fingers. The handpose model is provided by tensorflow.js and is able to detect 21 different distinct points in your hand. The model is a lightweight ML pipeline consisting of two models: A palm detector and a hand-skeleton finger tracking model. It predicts 21 3D hand key-points per detected hand.

This tutorial will show our readers how to build our very own Handpose detector using mediapipe and Python. 

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader will be able to:
1. Learn how to install and setup the pre-trained handpose model
2. Learn how to build the handpose application using a video camera/webcam
3. Learn how to detect these landmarks on the hand in real-time using the camera

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [MediaPipe HandPose](https://github.com/tensorflow/tfjs-models/tree/master/handpose)
- [Create a New React App](https://reactjs.org/docs/create-a-new-react-app.html)

### Article quality

How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

Tensorflow.js has many pre-trained models. The handpose model is one of their latest releases. Not many articles exist on the subject. Those that are present are either too complex to understand. We need to democratize AI and the use of such amazing pre-trained models should be understood with only a few lines of code. My tutorial aims to do just that. Keeping it simple but still unique.

### PR DESCRIPTION
**Pull request by smkinyan**
This closes #4794

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Why did you delete our template before submitting your PR? @smkinyan

**Comment by smkinyan** [comment]
> Why did you delete our template before submitting your PR? @smkinyan

@ahmadmardeni1 I did not delete it. I actually thought you guys had changed it or rather it was a problem on your side. Because when I clicked on `Create PR`, that's how it appeared.

File: content/articles/handpose-detection-using-mediapipe-and-python/index.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+### Installing and importing dependencies
+We perform a quick pip install for two core dependencies; the MediaPipe and openCV python libraries. MediaPipe is an open-sourced, cross-platform library that provides many ready-to-use ML solutions for solving computer vision problems. A few examples include ML solutions for [Face Detection](https://google.github.io/mediapipe/solutions/face_detection.html), [Selfie Segmentation](https://google.github.io/mediapipe/solutions/selfie_segmentation.html), [Hair Segmentation](https://google.github.io/mediapipe/solutions/hair_segmentation.html), and [Object Detection](https://google.github.io/mediapipe/solutions/object_detection.html). 
+For this tutorial, we will leverage the library to import the [MediaPipe Hands](https://google.github.io/mediapipe/solutions/hands) model in our project.
+
+We will also be installing the OpenCV library. Like MediaPipe, OpenCV is a library that also helps in solving computer vision problems. In this tutorial, we will use the library to work with images and access our webcam in real-time easily.
+
+```python
+!pip install mediapipe opencv-python
+```
```
Comment (collins-a) [inline content/articles/handpose-detection-using-mediapipe-and-python/index.md:50]: Switch to bash

**Review by collins-a — COMMENTED** [review]
Good tutorial on HandPose Detection. Handle the single revision and we'll be good to go.

**Comment by smkinyan** [comment]
@smkinyan done!

**Review by collins-a — APPROVED** [review]

**Comment by hectorkambow** [comment]
👍 overall nice work - very detailed and straight forward tutorial 
 
Please review and suggestions/changes made by your peer reviewer and/or myself during a final review.

**Pull request merged**